accounts/admin.py

```diff
@@ -16,6 +16,9 @@
 
 from django.contrib import admin
 from accounts.models import UserProfile
+from code_editor.models import File, Project
 
 # Register your models here.
 admin.site.register(UserProfile)
+admin.site.register(Project)
+admin.site.register(File)
```

Comment:
These models can be registered for the admin interface from code_editor/admin.py. That way the code might be a little more organized, but it works perfectly fine this way too.